scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -800,7 +977,85 @@
         [filter_second]
             side=1
         [/filter_second]
-        {MSG_narrator _"How did you manage to kill Uri van Roe? This shouldn't happen... it means the level is too easy. Mail adanilecki @ cs put poznan pl with your savegame and replay for review!"}
+#ifdef DEBUG_MODE
+        #po: Uri van Roe shouldn't die either way, whether debug mode is on or not, so the fact that we recognize debug mode specially here is just an easter egg:
+        {MSG_narrator _"Using debug mode to kill Uri van Roe is cheating!"}
+        [endlevel]
+            result=defeat
+        [/endlevel]
+#else
+        {MSG_narrator _"How did you manage to kill Uri van Roe? This shouldn't happen... it means the level is too easy. Stop what you are doing and mail adanilecki @ cs put poznan pl with your savegame and replay for review!"}
```

Comment:
Here too, just use the snide meta comment if you like, but don't actually alter the gameplay.

Reply:
ok, I cut back the gameplay-altering bits in 61ba872